co-noir/co-ultrahonk/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+pub(crate) mod co_decider;
+pub(crate) mod co_oink;
+pub(crate) mod parse;
+pub mod prelude;
+pub(crate) mod prover;
+pub(crate) mod types;
+
+use ark_ec::pairing::Pairing;
+use mpc_core::{
+    protocols::plain::PlainDriver,
+    traits::{EcMpcProtocol, MSMProvider, PrimeFieldMpcProtocol},
+};
+use parse::builder_variable::SharedBuilderVariable;
+use ultrahonk::prelude::ProverCrs;
+
+impl<P: Pairing> SharedBuilderVariable<PlainDriver<P::ScalarField>, P> {
+    pub fn promote_public_witness_vector(
+        witness: Vec<P::ScalarField>,
+    ) -> Vec<SharedBuilderVariable<PlainDriver<P::ScalarField>, P>> {
+        witness
+            .into_iter()
+            .map(SharedBuilderVariable::Public)
+            .collect()
+    }
+}
+
+pub(crate) const NUM_ALPHAS: usize = ultrahonk::NUM_ALPHAS;
+// The log of the max circuit size assumed in order to achieve constant sized Honk proofs
+// TODO(https://github.com/AztecProtocol/barretenberg/issues/1046): Remove the need for const sized proofs
+pub(crate) const CONST_PROOF_SIZE_LOG_N: usize = ultrahonk::CONST_PROOF_SIZE_LOG_N;
+pub(crate) const N_MAX: usize = ultrahonk::N_MAX;
+
+// TODO do not forget to remove this
```

Comment:
shoudl this be removed?

Reply:
This is just a comment for the merge with Francos rewrite, since FieldShareVec will not be there anymore